ospf6d/ospf6_neighbor.c

```diff
@@ -922,7 +949,8 @@ static void ospf6_neighbor_show(struct vty *vty, struct ospf6_neighbor *on,
 	} else
 		vty_out(vty, "%-15s %3d %11s %8s/%-12s %11s %s[%s]\n",
```

Comment:
Is this enough/aligned with the new output?

Reply:
in fact only json part is modified